doc/index.rst

```diff
@@ -6,10 +6,45 @@
 Welcome to Sphinx-Gallery's documentation!
 ==========================================
 
-Sphinx extension for automatic generation of an example gallery.
+A Sphinx extension that builds an HTML version of any python
+script and puts it into an examples gallery.
+
 It is extracted from the scikit-learn project and aims to be an
 independent general purpose extension.
 
+Why do we need it?
+------------------
+
+* Because we have a lot of code that does something good enough for
+  someone to learn about it.
+* Looking to the visual output is simpler to our minds than reading
+  the source code.
+* The Gallery workflow:
+   * Search for something that more or less looks to what you want
+   * Learn about it
+
+How does it look like
+---------------------
+
+You can structure your gallery from your python scripts and embed it
+in any place of your website. And it look like the next sample.
```

Comment:
Suggested rephrasing (for the whole paragraph above):

Here is an example gallery generated from a few Python scripts.